docs/release-notes/data-releases.md

```diff
@@ -4,20 +4,42 @@ title: Data Releases
 platform_key: DOCS_DATA_RELEASES
 ---
 
-A DCC data release is a curated data set produced from clinical and experimental data submitted through the ARGO Data Platform. Data releases happen on average four times a year and are cumulative in nature.
+A DCC data release is a curated data set produced from clinical and molecular data submitted to the ARGO Data Platform. Data releases happen on average four times a year and are cumulative in nature. Released data can be examined using the [File Repository](https://platform.icgc-argo.org/repository) and [downloaded](/docs/data-access/data-download) using the download client, provided that access to controlled data has been confirmed. To access controlled data, please see the [DACO application process here](/docs/data-access/data-access), or [apply for DACO approval here](https://icgc.org/daco).
 
-## Data Release 1.0
+<!---
+## Data Release 2.0
 
 **Release Date:**
 
 ### New Updates
 
-Coming soon.
 
 ### Bug Fixes
 
-Coming soon.
+None to report.
 
 ### Known Issues
 
-Coming soon.
+None to report.
+------>
+
+## Initial Data Release 1.0
+
+**Release Date:** June 19, 2020
+
+### New Updates
+
+ICGC ARGO is excited to announce its initial data release including data from 2 newly released programs [PACA-CA](https://www.icgc-argo.org/page/96/panchope) and [OCCAMS-GB](https://www.icgc-argo.org/page/112/occams). While both of these programs were members of the original ICGC-25K data portal, a selection of their data has been reprocessed against the latest GRCH38 reference genome using the [ARGO DNA Seq Pipeline](/docs/analysis-workflows/harmonization-dna-pipeline). This release includes:
```

Comment:
change "GRCH38 reference genome" to 
`GRCh38 Human Reference Genome`

Reply:
also check this url...it may be "docs/analysis-workflows/dna-pipeline" ?

Reply:
no i think keep the focus on argo here